client/layout/guided-tours/design-showcase-tour.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import {
+	overEvery as and,
+	negate as not,
+} from 'lodash';
+import { isDesktop } from 'lib/viewport';
+
+import {
+	makeTour,
+	Tour,
+	Step,
+	ButtonRow,
+	Next,
+	Quit,
+	Continue,
+} from 'layout/guided-tours/config-elements';
+import {
+	inSection,
+	isNewUser,
+	isEnabled,
+	selectedSiteIsCustomizable,
+} from 'state/ui/guided-tours/contexts';
+import Gridicon from 'components/gridicon';
+
+const isAbTestInVariant = function() {};
+
+export const DesignShowcaseTour = makeTour(
+	<Tour
+		name="designShowcase"
+		version="20161123"
+		path="/design"
+		when={ and(
+			isNewUser,
+			isEnabled( 'guided-tours/design-showcase' ),
+			isDesktop,
+			selectedSiteIsCustomizable,
+			not( inSection( 'customize' ) ),
+			isAbTestInVariant( 'designShowcaseTour', 'enabled' ),
+			) }
```

Comment:
Minor: should we place the feature flag and A/B test checks first in the `and` list? For consistency.